Atypon Journals.js

```diff
@@ -720,441 +720,6 @@ var testCases = [
 						"creatorType": "author",
 						"fieldMode": 1
 					},
-					{
```

Comment:
This doesn't seem right - where'd all the authors go?

Reply:
All those authors are part of the COVID-19 Review Consortium. They're not listed at all in the RIS etc. export and they're included in the same content attribute as COVID-19 Review Consortium in the metatags with some quite wonky HTML. I don't think we should try to fix it (even in terms of what _should_ be in the metadata it's a borderline case)

Reply:
Makes sense.